src/repositories/QueueRepository/__tests__/QueueRepository.int.spec.ts

```diff
@@ -208,4 +209,48 @@ describe("Queue Repository tests", () => {
     expect(DateTime.fromJSDate(playerInDb?.queueTime!).toISO()).toEqual(mockBallChaser.queueTime.toISO());
     expect(playerInDb?.team).toBeNull();
   });
+  describe("check if player is in queue", () => {
+    it("player is in queue", async () => {
+      const mockBallChaser = BallChaserQueueBuilder.single();
+      await manuallyAddBallChaserToQueue(mockBallChaser);
+
+      const playerInQueue = await QueueRepository.isPlayerInQueue(mockBallChaser.id);
+
+      expect(playerInQueue).toEqual(true);
+    });
+    it("player is not in queue", async () => {
+      const mockBallChaser = BallChaserQueueBuilder.single();
+      await manuallyAddBallChaserToQueue(mockBallChaser);
+
+      const playerNotInQueue = await QueueRepository.isPlayerInQueue(faker.datatype.uuid());
+
+      expect(playerNotInQueue).toEqual(false);
+    });
+  });
+  describe("check if player is team captain", () => {
+    it("player is team captain", async () => {
+      const mockBallChaser = BallChaserQueueBuilder.single({ team: Team.Blue, isCap: true });
+      await manuallyAddBallChaserToQueue(mockBallChaser);
+
+      const isCaptain = await QueueRepository.isTeamCaptain(mockBallChaser.id, Team.Blue);
+
+      expect(isCaptain).toEqual(true);
+    });
+    it("player is team captain but wrong team", async () => {
+      const mockBlueCaptain = BallChaserQueueBuilder.single({ team: Team.Blue, isCap: true });
+      await manuallyAddBallChaserToQueue(mockBlueCaptain);
+
+      const isCaptain = await QueueRepository.isTeamCaptain(faker.datatype.uuid(), Team.Orange);
```

Comment:
Shouldn't this be `mockBlueCaptain.id` instead of `faker.datatype.uuid()`?

Reply:
Fixed in next commit.